webpack/assets/javascripts/react_app/components/about/index.js

```diff
@@ -0,0 +1,25 @@
+
+import React from 'react';
+import ForemanTabs from '../common/tabs';
+import AboutComputeTable from './compute';
+import AboutPluginTable from './plugin';
+import AboutProviderTable from './provider';
+import AboutProxyTable from './proxies';
+
+class About extends React.Component {
+  render() {
+    const {
+      compute, proxy, provider, plugin,
+    } = this.props.data;
+    const tabs = [__('Smart Proxies'), __('Available Providers'), __('Compute Resources'), __('Plugins')];
+
+    return <ForemanTabs id='about_tabs' tabs={tabs} >
```

Comment:
this looks really close to how react-router works.. I wonder if we should combine the two? (so if you refresh you come back to the same tab)